config/config_encryption.go

```diff
@@ -130,12 +131,18 @@ func (c *Config) encryptConfigData(configData []byte) ([]byte, error) {
 
 	ciphertext := aead.Seal(nil, nil, configData, nil)
 
-	appendedFile := append(bytes.Clone(encryptionPrefix), c.storedSalt...)
-	appendedFile = append(appendedFile, encryptionVersionPrefix...)
-	versionOffset := len(encryptionPrefix) + len(c.storedSalt) + len(encryptionVersionPrefix)
-	appendedFile = append(appendedFile, make([]byte, 2)...)
-	binary.BigEndian.PutUint16(appendedFile[versionOffset:], encryptionVersion)
-	appendedFile = append(appendedFile, ciphertext...)
+	totalLen := len(encryptionPrefix) + len(c.storedSalt) + len(encryptionVersionPrefix) + versionSize + len(ciphertext)
+	appendedFile := make([]byte, totalLen)
```

Comment:
I'm easy, but arguably:
```suggestion
	appendedFile := make([]byte, len(encryptionPrefix) + len(c.storedSalt) + len(encryptionVersionPrefix) + versionSize + len(ciphertext))
```
For me, it's very implicit that the maths there amounts to "total length"